src/plugins/home/public/application/components/move_data/move_data.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, MouseEvent } from 'react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiImage,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { METRIC_TYPE, UiCounterMetricType } from '@kbn/analytics';
+import { i18n } from '@kbn/i18n';
+import { createAppNavigationHandler } from '../app_navigation_handler';
+
+interface Props {
+  addBasePath: (path: string) => string;
+  isCloudEnabled: boolean;
+  trackUiMetric: (type: UiCounterMetricType, eventNames: string | string[], count?: number) => void;
+}
+
+export const MoveData: FC<Props> = ({ addBasePath, isCloudEnabled, trackUiMetric }) => {
+  const migrateDataUrl = isCloudEnabled
+    ? 'https://ela.st/cloud-migration'
+    : '/app/management/data/migrate_data';
+  const buttonLabel = (
+    <FormattedMessage
+      id="home.addData.moveYourDataButtonLabel"
+      defaultMessage="Explore the benefits"
+    />
+  );
+
+  return (
+    <EuiPanel paddingSize="l">
+      <EuiFlexGroup alignItems="center" gutterSize="xl">
+        <EuiFlexItem>
+          <EuiImage
+            alt={i18n.translate('home.moveData.illustration.alt.text', {
+              defaultMessage: 'Illustration for cloud data migration',
+            })}
+            src={addBasePath('/plugins/kibanaReact/assets/') + 'illustration_cloud_migration.png'}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiTitle size="xs">
+            <h4>
+              <FormattedMessage
+                id="home.addData.moveYourDataTitle"
+                defaultMessage="Considering Elastic Cloud?"
+              />
+            </h4>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText size="s">
+            <FormattedMessage
+              id="home.addData.moveYourDataToElasticCloud"
+              defaultMessage="Moving your data to Elastic Cloud is easy and can save you time and money."
+            />
+          </EuiText>
+          <EuiSpacer size="m" />
+          {isCloudEnabled ? (
```

Comment:
As discussed in our sync earlier today, the ECMA link is not intended to display on cloud. What https://github.com/elastic/kibana/issues/147462 was trying to address is the situation where a user has limited ES privileges and does not have access to Stack Management. I think we should go back to the original plan and add the docs link to the help menu for self-managed users, as the help menu will always be displayed regardless of privileges. We can include the docs team for help with the copy for the link.